docs/how-tos/howto-query-operator.md

```diff
@@ -12,6 +12,8 @@ A functioning node is required in order to query any kind of operator informatio
 With the Polygon SDK, node operators are in control and informed about what the node they're operating is doing.<br />
 At any time, they can use the node information layer, built on top of gRPC, and get meaningful information - no log sifting required.
 
+**Note:** If your node isn't running on `127.0.0.1:8545` you should add a flag `--grpc-address <address:port>` to the commands listed in this document.
```

Comment:
You can add the note like this, to make it stand out more:
https://github.com/0xPolygon/polygon-sdk-docs/blob/main/docs/overview.md

With :::note

Reply:
But it isn't parsed by the markdown, I just see three colons and `note` 🤷 
<img width="444" alt="Screen Shot 2021-06-04 at 14 32 33 PM" src="https://user-images.githubusercontent.com/10708079/120801664-bc70fe80-c541-11eb-946e-cf189a2fe673.png">

Reply:
We use this trick from docusaurus:
https://docusaurus.io/docs/markdown-features/admonitions

Reply:
I see, will do that. Thanks! 🍻